lib/theme.js

```diff
@@ -36,7 +36,7 @@ const theme = createMuiTheme({
       main: red.A400,
     },
     background: {
-      default: '#fff',
+      default: '#e5e5e5',
```

Comment:
I see why you are using `#e5e5e5`, it's probably a bug in Figma. I think`#f5f5f5` is the correct one.

![bg](https://user-images.githubusercontent.com/108608/79692286-d3e61d00-8296-11ea-8259-2b90de25d953.gif)